dotbot/protocol.py

```diff
@@ -14,17 +14,17 @@
 class PayloadType(Enum):
     """Types of DotBot payload types."""
 
-    CMD_MOVE_RAW = 0
-    CMD_RGB_LED = 1
-    LH2_RAW_DATA = 2
-    LH2_LOCATION = 3
-    ADVERTISEMENT = 4
-    GPS_POSITION = 5
-    DOTBOT_DATA = 6
-    CONTROL_MODE = 7
-    LH2_WAYPOINTS = 8
-    GPS_WAYPOINTS = 9
-    SAILBOT_DATA = 10
+    CMD_MOVE_RAW    = 0
```

Comment:
you don't need to align. That creates unrelated changes, making the review harder.